CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing
```

Comment:
@MidouWebDev Is this file supposed to be here? Things should be changed to be api specific.

Reply:
@j-dogcoder i modified it to be specific for the api repo !